collection-scripts/gather_ctlplane_resources

```diff
@@ -28,14 +28,17 @@ for p in $(oc -n "$NS" get pods -o custom-columns=":metadata.name"); do
     echo "Dump logs for pod: $p";
     mkdir -p "${NAMESPACE_PATH}"/"$NS"/pods/"$p"/logs
     # describe pod
-    /usr/bin/oc -n "$NS" describe pod "$p" > "${NAMESPACE_PATH}"/"$NS"/pods/"$p"/"$p"-describe.log
-    # get logs
-    /usr/bin/oc -n "$NS" logs --prefix=true --all-containers=true "$p" > "${NAMESPACE_PATH}"/"$NS"/pods/"$p"/logs/"$p".log;
+    /usr/bin/oc -n "$NS" describe pod "$p" > "${NAMESPACE_PATH}/${NS}/pods/${p}/${p}-describe"
+    # get logs for each of the individual containers
+    containers=`/usr/bin/oc -n $NS get pod $p -o jsonpath='{.spec.containers[*].name}'`
```

Comment:
awesome, it makes sense now that we have sidecar containers everywhere. I had this change in the backlog and I think it's cleaner having a log file for each container and get the "previous" log if there is one.